docs/design/services/dynamodb/high-level-library/README.md

```diff
@@ -0,0 +1,38 @@
+# Project Tenets (unless you know better ones)
+
+1. Meeting customers in their problem space allows them to deliver value
+   quickly.
+2. Meeting customer expectations drives usability.
+3. Discoverability drives usage.
+
+# Introduction
+
+This project provides a Dynamo DB experience that is easier to use and
+understand than can be provided by the generated Dynamo DB client. It
+provides object-oriented features, like persisting and updating Java
+data objects, as well as expressive data retrieval operations.
+
+# Project Goals
```

Comment:
Do we have goals on what operations/features from low-level clients that the high level client supports?
I mean: Should all features supported in low-level client be supported in high level client (or) a subset (if so, how do we determine the subset)

Reply:
"The high-level library should be the ideal way to integrate with Dynamo DB." as a goal seems to indicate to me that we should support all features in the low-level client. i.e. people shouldn't want to use the low level client.